### ISSUE TITLE
the class center distance expansion loss of FastMLDNN

### ISSUE THREAD

**Issue by Patrick-Nick** [Singingkettle/ChangShuoRadioRecognition#5]
I have read your paper about FastMLDNN. This is really excellent work. When I looked up your code, it seemed that I couldn't find the code of the class center distance expansion loss. Perhaps I missed it?

**Comment by Singingkettle** [comment]
Hi, recently, the code about FastMLDNN is rewritten upon new mmengine. Basically, the  class center is made up of two parts in python code: class center distance and loss function (cross entropy loss ).
![image](https://github.com/Singingkettle/ChangShuoRadioRecognition/assets/14244102/b16990bf-b995-4353-aa46-5f50c20ab037)

 The class distance is calculated in code (csrr/models/backbones/fastmldnn.py):

"p = sim_matrix(self.classifier[3].weight, self.classifier[3].weight)"

The loss is cross entropy loss in fast_mldnn_head.py

**Comment by asq12138** [comment]
In the newly updated code, I can't find the fastmldnn.py file. And in (csrr/models/backbones/__init__.py), I can't find 'FastMLDNN' being registered.

**Comment by asq12138** [comment]
In the newly updated code, I can't find the fastmldnn.py file(csrr/models/backbones/fastmldnn.py). And in (csrr/models/backbones/init.py), I can't find 'FastMLDNN' being registered.

**Comment by asq12138** [comment]
It's really excellent work. But I can't find the file that contains the sim_matrix function, and the fastmldnn.py file in current version.
such as, from ...ops import sim_matrix, but i can't Correct import.
